clarifai.go

```diff
@@ -37,7 +37,7 @@ func NewClarifaiClient() *ClarifaiClient {
 }
 
 // Check whether the client still has access rights to Clarifai
-func (client *ClarifaiClient) isAccess() bool {
+func isCLFAccessible(client *ClarifaiClient) bool {
```

Comment:
hmm yeah I guess I didn't think about it that much. You're right, it doesn't make much of a difference. Appending these methods to a struct makes a lot more sense to me, but maybe that's because I mostly did OO up until now. I guess it comes down to personal preference.